PowerEditor/installer/nativeLang/polish.xml

```diff
@@ -10,7 +10,7 @@ Translation note:
 	- Updated by Patryk Skorupa (ppskorupa@outlook.com) to version 7.7.2 (29/07/2019)
 	- Updated by Cezariusz Marek to version 7.8.3 (11.01.2020)
 	- Updated by Piotr Kostrzewski (piotrkostrzewski2@@outlook.com) to version 8.7.2 (27.11.2024)
-	- Updated by Arkadiusz Michalski (webref.pl) to version 8.8.1 (25.06.2025)
+	- Updated by Arkadiusz Michalski (webref.pl) to version 8.8.4 (02.08.2025)
```

Comment:
For the PR of translation, don't guess or use the next version number. Use the current version number instead.